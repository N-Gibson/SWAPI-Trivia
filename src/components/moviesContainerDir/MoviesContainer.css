.movie-container {
  display: grid;
  grid-template-columns: 1fr 1fr 1fr;
  grid-column-gap: 20px;
  /* margin: 40px; */
  min-height: 100vh;
  padding: 40px;
}
