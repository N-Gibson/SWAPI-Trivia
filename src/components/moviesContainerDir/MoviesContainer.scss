.movie-container {
  display: grid;
  grid-template-columns: auto auto auto;
  grid-gap: 10%;
  height: 100%;
  padding: 30px;
  justify-content: center;
  overflow: scroll;
}
