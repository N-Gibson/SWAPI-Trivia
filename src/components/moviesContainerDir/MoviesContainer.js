import React from 'react'
import './MoviesContainer.css'
import MovieCard from '../movieCardDir/movieCard'


const MoviesContainer = (props) => {
  console.log("props.movies[0]", props.movies[0])
  return (
    <section className='movie-container'>
      {props.movies.map(movie => {
        console.log(movie.image)
        return <MovieCard 
          image={movie.image}
          title={movie.title}
          releaseDate={movie.release_date}
          id={movie.episode_id}
          key={movie.episode_id}
        />
      })}
    </section>
  )
}

export default MoviesContainer