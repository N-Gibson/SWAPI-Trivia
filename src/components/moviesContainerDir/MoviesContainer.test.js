import React from 'react';
import { shallow } from 'enzyme';
import MoviesContainer from './MoviesContainer';

describe('Character Container', () => {
  let wrapper
  beforeEach(() => {

  })

  it('', () => {
    
  })
})