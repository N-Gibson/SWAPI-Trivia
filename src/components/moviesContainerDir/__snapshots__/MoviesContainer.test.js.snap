// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviesContainer should match snapshot 1`] = `
<Fragment>
  <Nav />
  <section
    className="movie-container"
  >
    <MovieCard
      id={1}
      image="imageUrl"
      key="1"
      releaseDate="2019-01-01"
      title="Movie Title"
    />
    <MovieCard
      id={1}
      image="imageUrl"
      key="1"
      releaseDate="2019-01-01"
      title="Movie Title"
    />
  </section>
</Fragment>
`;
